censys/asm/client.py

```diff
@@ -0,0 +1,146 @@
+"""
+Base for interacting with the Censys Search API.
+"""
+# pylint: disable=too-many-arguments
+import os
+
+from math import inf
+from typing import Type, Dict, Generator, Optional
+from requests.models import Response
+from censys.exceptions import *
```

Comment:
Could we make a new `censys.asm.exceptions` class and have the mapping done there to avoid `*` imports?